interchaintest/helpers/cosmwasm.go

```diff
@@ -92,3 +92,91 @@ func ExecuteMsgWithFee(t *testing.T, ctx context.Context, chain *cosmos.CosmosCh
 		t.Fatal(err)
 	}
 }
+
+func ExecuteAuthzGrantMsgWithFee(t *testing.T, ctx context.Context, chain *cosmos.CosmosChain, granter ibc.Wallet, grantee ibc.Wallet, contractAddr, amount, feeCoin, message string) {
```

Comment:
Future PR but we should abstract these away from hardcoded msg types, but out of scope here

Reply:
update nvm I did it now as there were some unused args already